src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,32 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.CustomException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomException {
+        if (user == null) {
+            throw new CustomException("User is null.");
+        } else if (user.getLogin() == null
+                || user.getPassword() == null
+                || user.getAge() == null) {
+            throw new CustomException("The parameters of user are null.");
```

Comment:
The exception message 'The parameters of user are null.' is not informative enough. Specify which parameter is null for better clarity.